pages/convert9To7.js

```diff
@@ -0,0 +1,219 @@
+import {
+  useContext, useEffect, useState,
+} from 'react'
+import { useRouter } from 'next/router'
+import Button from '@material-ui/core/Button'
+// import CircularProgress from '@material-ui/core/CircularProgress'
+import {convertTsv9to7} from '@utils/tsvConversion';
+
+import { AuthenticationContext } from 'gitea-react-toolkit'
+import Layout from '@components/Layout'
+import { StoreContext } from '@context/StoreContext'
+import { AdminContext } from '@context/AdminContext'
+import { tCCreateBranchesExist, 
+  createArchivedTsv9Branch, 
+  saveNewTsv7, updateManifestWithProjects } from '@utils/dcsApis'
+import { doFetch } from '@utils/network'
+import { decodeBase64ToUtf8 } from '@utils/decode'
+import { TN } from '@common/constants';
+
+// import { createRelease } from '@utils/dcsApis'
+// import Link from '@material-ui/core/Link'
+
+const ConvertPage = () => {
+  const router = useRouter()
+  const [convertMessages, setConvertMessages] = useState([])
+  const [disableConvert, setDisableConvert] = useState(true)
+  const [confirmConvert, setConfirmConvert] = useState(false)
+
+  const { state: authentication } = useContext(AuthenticationContext)
+  const {
+    state: {
+      owner: organization,
+      languageId,
+      server,
+    },
+  } = useContext(StoreContext)
+
+  const {
+    state: { 
+      tnRepoTree,
+      tnRepoTreeManifest,
+      tnManifestSha,
+    },
+    actions: {
+      setRefresh,
+    }
+  } = useContext(AdminContext)
+
+  // archive the existing branch for posterity
+  // then start converting
+  useEffect( () => {
+    async function archiveBranch () {
```

Comment:
This function seems misnamed.
```suggestion
    async function performConversion () {
```